essence-of-live-coding-vivid/src/LiveCoding/Vivid.hs

```diff
@@ -39,8 +42,18 @@ vividHandleParametrised = ParametrisedHandle { .. }
     changeParametrised old new synth = defaultChange createParametrised destroyParametrised old new synth
 
 deriving instance Eq (SynthDef args)
+deriving instance Data SynthState
+deriving instance KnownSymbol a => Data (I a)
 
 liveSynth
```

Comment:
@miguel-negrao Do you have a good usage example for this function?

Reply:
All examples in eolc-vivid-midi-demo (in eolc-midi-demo/app/) use this function except for 
SC_Embedded.hs.